dash_app/dashboard.py

```diff
@@ -55,7 +62,62 @@ def _generate_next_value_(name, start, count, last_values):  # pylint: disable=n
     REGION_GRAPH = enum.auto()
     REGION_TAG_TABLE = enum.auto()
     DATATABLE_REGIONS = enum.auto()
-    # TODO(tom): Add remained of dash ids to this enum.
+    DATASETS_DROPDOWN = enum.auto()
+    REGIONS_VARIABLE_DROPDOWN = enum.auto()
+    PIVOT_TABLE_PRESETS_DROPDOWN = enum.auto()
+
+
+@enum.unique
+class DashboardFile(GetByValueMixin, str, enum.Enum):
+    """Enum of files that may be displayed in the dashboard."""
+
+    # Define a custom __new__ so DashboardFile instances use 'filename' as their value and have a
+    # description.
+    def __new__(cls, filename, description):
+        # Initialize super class (str) with filename.
+        obj = super().__new__(cls, filename)
+        # _value_ is a special name of enum. Set it here so enum code doesn't attempt to call
+        # str(filename, description).
+        obj._value_ = filename
+        obj.description = description
+        obj.filename = filename
+        return obj
+
+    COMBINED_DATA = "multiregion-wide-date", "Combined data"
+    MANUAL_FILTER_REMOVED = "manual_filter_removed", "Manual filter removed timeseries"
+
+
+@dataclasses.dataclass(frozen=True)
+class RepoWrapper:
+    _repo: git.Repo
+    head_commit_str: str
+    _working_copy_dataset: timeseries.MultiRegionDataset
+
+    @staticmethod
+    def make() -> "RepoWrapper":
+        # Add odbt=git.db.GitDB if there are problems reading files from git. See
+        # https://github.com/gitpython-developers/GitPython/issues/642#issuecomment-618117030
```

Comment:
Not sure how likely it is that people will hit this, but if it's at all likely, then it might be nice to catch the problem if at all possible and either automatically apply the work around or log an explicit message about the workaround, so somebody doesn't have to go spelunking to find this comment.   But if it's not likely, then meh. 🤷